calendar-service/calendar-service.sh

```diff
@@ -50,7 +50,7 @@ start() {
   done < "$loginFile"
 
   # Redirects default and error output to a log file
-  java -Dpa.scheduler.home=$cwd -Dspring.config.location=$cwd/config/calendar-service/application.properties -jar $cwd/tools/calendar-service*.jar > /dev/null 2>&1 &
+  java -Dpa.scheduler.home=$cwd -Dspring.config.location=$cwd/config/calendar-service/application.properties -jar $cwd/calendar-service/calendar-service*.jar > /dev/null 2>&1 &
```

Comment:
all paths like $cwd  , $cwd/config/calendar-service , etc should be between double-quotes to avoid problems when the path contains spaces